File: Scripts/generated_codes/java/22.java
"""
"""

import java.util.*;
public class Main {
    public static void main(String args[]) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        int firstDigit, lastDigit;
        while (n-->0){
            int num = sc.nextInt();
            firstDigit=num%10;
            while(num>9)
            {
                num=num/10;
            }
            lastDigit=num;
            System.out.println(firstDigit+lastDigit);
        }
    }
}

File: Scripts/generated_codes/python/codechef_responses/22.py
'''

'''

def sumOfFirstLast(n) :
    last_digit = n % 10
    first_digit = n
    while n >= 10 :
        n = n//10
    first_digit = n
    return first_digit + last_digit
    
t = int(input())
for i in range(t) :
    n = int(input())
    print(sumOfFirstLast(n))